dockers/docker-dhcp-relay/config.sh

```diff
@@ -1,4 +1,12 @@
 #!/bin/bash
 
-sonic-cfggen -m /etc/sonic/minigraph.xml -y /etc/sonic/dhcp_relay.yml -t /usr/share/sonic/templates/isc-dhcp-relay.j2 > /etc/default/isc-dhcp-relay
+MINIGRAPH_HOSTNAME=`sonic-cfggen -m /etc/sonic/minigraph.xml -v "minigraph_hostname"`
+DEVICE_ROLE=`sonic-cfggen -m /etc/sonic/minigraph.xml -v "minigraph_devices['$MINIGRAPH_HOSTNAME']['type']"`
+
```

Comment:
Good to know. Simplified in https://github.com/Azure/sonic-buildimage/pull/371/commits/265311e7af283ec8c62bb950ccfc348dcfa787d2. Thanks!